db/migrate/20170306042925_associate_affiliate_with_coordinators.rb

```diff
@@ -0,0 +1,6 @@
+class AssociateAffiliateWithCoordinators < ActiveRecord::Migration[5.0]
+  def change
+    add_column :coordinators, :affiliate_id, :integer
+    add_index :coordinators, :affiliate_id
```

Comment:
`add_index` will enforce uniqueness between these two columns, such that we can never have two rows with the same coordinator and affiliate. I don't see a reason we necessary need to have this, but I think it only stands to pose a problem down the line, especially for the other models, since we aren't concerned about accidentally duplications and we *may* have duplicated entries in the event that people move around and come back.